packet/packet_composed/src/composed.cpp

```diff
@@ -0,0 +1,109 @@
+#include <chrono>
+#include <functional>
+#include <string>
+#include "packet_composed/sensorspacket.hpp"
+
+
+Composed::Composed() :
+    rclcpp::Node("composed"),
+    _publisher(),
+    _timer(),  
+    _count(0)  
+{
+    //composed_publisherをコンストラクタで初期化した。
+    using namespace std::chrono_literals;
+    _publisher = this->create_publisher<std_msgs::msg::String>("/sensors_composed", 10);
+    composed_publisher = this->create_publisher<packet_interfaces::msg::Composed>("composed_topic", 10);
```

Comment:
stringのpublishはしません & topic名はFigmaのやつ(から`/packet/`のプレフィックスを取り除いたもの)と合わせるようにお願いします

```suggestion
    composed_publisher = this->create_publisher<packet_interfaces::msg::Composed>("sensors_composed", 10);
```

Reply:
![image](https://github.com/rogy-AquaLab/2024_cavolinia/assets/145688154/c438b5cc-8bfa-4aff-b44e-5003be849fd6)
voltageの部分なら、`packets/sensors/voltage`で`voltage`と、topic名を書いてみたのですが、あってますか？

Reply:
この部分変えてなかったです。topic名変えました。